acme_diags/plot/cartopy/area_mean_time_series_plot.py

```diff
@@ -1,34 +1,41 @@
 import os
-import matplotlib
-matplotlib.use('agg')
-import matplotlib.pyplot as plt
+
 import numpy as np
+
 from acme_diags.driver.utils.general import get_output_dir
 
+import matplotlib  # isort:skip
```

Comment:
Why does this need to be skipped if line 10 is the only one that has to come after `matplotlib.use("agg")`?

Reply:
I originally had it skipped so the `matplotlib `imports were all grouped together, but now realize it is actually unnecessary. I'll remove the skips for the main `import matplotlib` since that is always above the rest of the `matplotlib` stuff anyways.